src/bp/core/services/auth/auth-service.ts

```diff
@@ -131,7 +137,7 @@ export default class AuthService {
       last_logon: new Date()
     })
 
-    return generateUserToken(email, TOKEN_AUDIENCE)
+    return generateUserToken(email, TOKEN_AUDIENCE, await this.configProvider.getBotpressConfig())
```

Comment:
really not clear why config is passed as an object to the user token generation, should be avoided